inc/classes/class_auth.php

```diff
@@ -1,34 +1,102 @@
 <?php
 
 /**
- * Authorisation and Cookiemanagement for Lansuite
+ * Class auth
+ *
+ * Authorisation and Cookie management for LanSuite
  *
- * @package lansuite_core
- * @author bytekilla
- * @version $Id$
- * @access public
  * @todo Change uniqkey from md5(password) to an extra Field
  */
 class auth {
 
-    public $auth = array();                 // Userdaten im Array
-    public $timestamp;                      // Zeit
-    public $cookie_data =     array();      // Cookiedaten
-    public $cookie_name =     "LSAUTH";     // Cookiename
-    public $cookie_version =  "1";          // Cookieversion
-    public $cookie_domain =   "";           // Domain
-    public $cookie_time =     "30";         // Dauer in Tagen
-    public $cookie_path =     "";           // Cookiepath. Left blank for autodetect
-    public $cookie_crypt =    true;         // Crypt Cookie with AzDGCrypt
-    public $cookie_crypt_pw = "iD9ww32e";   // Passphrase for AzDGCrypt
-    public $online_users = array();         // Array containing all users, currently online
-    public $away_users = array();         // Array containing all users, currently online by ajax but no hit last 10min
-
-  /**
-   * CONSTRUCTOR : Initialize basic Variables for Authorisation
-   * @param mixed Frameworkmode for switch Stats
-   *
-   */
+    /**
+     * Userdata
+     *
+     * @var array
+     */
+    public $auth = [];
+
+    /**
+     * Time
+     *
+     * @var int
+     */
+    public $timestamp;
+
+    /**
+     * Cookie data
+     *
+     * @var array
+     */
+    public $cookie_data = [];
+
+    /**
+     * Cookie name
+     *
+     * @var string
+     */
+    public $cookie_name = 'LSAUTH';
+
+    /**
+     * Cookie version
+     *
+     * @var string
+     */
+    public $cookie_version = '1';
+
+    /**
+     * Domain
+     *
+     * @var string
+     */
+    public $cookie_domain = '';
+
+    /**
+     * Duration in days
+     *
+     * @var string
+     */
+    public $cookie_time = '30';
+
+    /**
+     * Cookie path
+     *
+     * @var string
+     */
+    public $cookie_path = '';
+
+    /**
+     * Crypt Cookie with AzDGCrypt
+     *
+     * @var bool
+     */
+    public $cookie_crypt = true;
+
+    /**
+     * Passphrase for AzDGCrypt
+     *
+     * @var string
+     */
+    public $cookie_crypt_pw = "iD9ww32e";
```

Comment:
Ahem.
This we should definitely change to an unique key per installation.
Not as a part of this, PR but in general